docs/api.md

```diff
@@ -1,17 +1,21 @@
 ---
-title: JSON API documentation
 layout: page
 permalink: /docs/api/
 ---
-## Formulae
-### List formulae metadata for all {{ site.taps.core.repo }} formulae
-List the `brew info --json=v1` output for all current {{ site.taps.core.fullname }} formulae.
+# JSON API documentation
```

Comment:
Good call. Does this set the page title too?

Reply:
Yes, courtesy of [jekyll-titles-from-headings](https://github.com/benbalter/jekyll-titles-from-headings), part of the [github-pages](https://pages.github.com/versions/) gem.